src/app/_services/editor-pref.service.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { QueryList } from '@angular/core';
+import { async, TestBed } from '@angular/core/testing';
+import { CodemirrorComponent } from 'ng2-codemirror';
+
+import { EditorPrefService } from '.';
+
+function makeQueryList(): QueryList<CodemirrorComponent> {
+  // tslint:disable: no-any
```

Comment:
please use `// tslint:disable-next-line: no-any` and put it in the line just above the `any`. if you use `tslint:disable` it will be disabled for the rest of the file.